src/Simplex/Chat/Remote.hs

```diff
@@ -554,11 +576,12 @@ verifyRemoteCtrlSession execChatCommand sessCode' = handleCtrlError "verifyRemot
 stopRemoteCtrl :: ChatMonad m => m ()
 stopRemoteCtrl = cancelActiveRemoteCtrl False
 
-handleCtrlError :: ChatMonad m => Text -> m a -> m a
-handleCtrlError name action = action `catchChatError` \e -> do
-  logError $ name <> " remote ctrl error: " <> tshow e
-  cancelActiveRemoteCtrl True
-  throwError e
+handleCtrlError :: (HasCallStack, ChatMonad m) => Text -> m a -> m a
+handleCtrlError name action =
+  action `catchChatError` \e -> do
+    withFrozenCallStack $ logError $ name <> " remote ctrl error: " <> tshow e
```

Comment:
what's that

Reply:
fixing provenance in log messages. `handleCtrlError` is used multiple times, but source position tells the message comes from `handleCtrlError` itself which is useless.

this actually obsoletes the `name` argument.